app/src/main/java/com/kelvin/bdaypost/birthday/data/model/Contact.kt

```diff
@@ -0,0 +1,66 @@
+package com.kelvin.bdaypost.birthday.data.model
+
+import timber.log.Timber
+
+data class Contact(var birthday: Birthday, var contactInfo: ContactInfo)
```

Comment:
Are you planning to have `Birthday` ever do more than store a date? If not, we could probably use java's built-in date class